bin/test.sh

```diff
@@ -2,6 +2,8 @@
 
 set -ex
 
+phpunit
```

Comment:
We should make this a part of the default package scaffold too: https://github.com/wp-cli/scaffold-package-command/issues/121